app/_hub/kong-inc/graphql-rate-limiting-advanced/index.md

```diff
@@ -398,10 +431,24 @@ Cost decoration is available on the following routes:
 For example:
 
 ```
-http -f :8001/services/example/graphql-rate-limiting-advanced/costs type_path="Query.allPeople" mul_arguments="first"
-http -f :8001/services/example/graphql-rate-limiting-advanced/costs type_path="Person.vehicleConnection" mul_arguments="first" add_constant=42
-http -f :8001/services/example/graphql-rate-limiting-advanced/costs type_path="Vehicle.filmConnection" mul_arguments="first"
-http -f :8001/services/example/graphql-rate-limiting-advanced/costs type_path="Film.characterConnection" mul_arguments="first"
+$ curl -X POST http://kong:8001/services/{service}/graphql-rate-limiting-advanced/costs \
+  --data type_path="Query.allPeople" \
+  --data mul_arguments="first"
+
+
+$ curl -X POST http://kong:8001/services/{service}/graphql-rate-limiting-advanced/costs \
+  --data type_path="Person.vehicleConnection" \
+  --data mul_arguments="first"
+  --data add_constant=42
+
+$ curl -X POST http://kong:8001/services/{service}/graphql-rate-limiting-advanced/costs \
+  --data type_path="Vehicle.filmConnection" \
+  --data mul_arguments="first"
+
+
+$ curl -X POST http://kong:8001/services/{service}/graphql-rate-limiting-advanced/costs \
+  --data type_path="Film.characterConnection" \
+  --data mul_arguments="first"
```

Comment:
In this example and the examples above, this topic uses three different syntax styles:

`--data "name=example"\`
`--data name=graphql-rate-limiting-advanced \`
`--data 'name=graphql-rate-limiting-advanced' \`

Any chance we can be more consistent?

Reply:
> In this example and the examples above, this topic uses three different syntax styles:
> 
> `--data "name=example"\`
> `--data name=graphql-rate-limiting-advanced \`
> `--data 'name=graphql-rate-limiting-advanced' \`
> 
> Any chance we can be more consistent?

@lena-larionova yes there is certainly a chance to be more consistent. Defining a style guide for preferred/recommended syntax should be done. Especially for array syntax as I saw mentioned in other channels that we need recommended and consistent syntax. I'm going to leave this for now because we don't have the bandwidth to test everything and I've already spent a week and a half on this PR.